operator_ui/__tests__/components/JobRuns/StatusCard.test.js

```diff
@@ -1,8 +1,32 @@
 import React from 'react'
 import StatusCard from 'components/JobRuns/StatusCard'
 import mountWithTheme from 'test-helpers/mountWithTheme'
+import isoDate from 'test-helpers/isoDate'
 
 describe('components/JobRuns/StatusCard', () => {
+  const minuteAgo = isoDate(Date.now() - 60)
```

Comment:
I believe this is 60ms. Can you use the `MINUTE_MS` & `TWO_MINUTE_MS` constants from the `isoDate` module